File: outgoing-simple-java-demo/src/main/java/de/codecentric/cm4sb/demo/outgoing/repository/MovieRepository.java
package de.codecentric.cm4sb.demo.outgoing.repository;

import de.codecentric.cm4sb.demo.outgoing.domain.Movie;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MovieRepository extends CrudRepository<Movie, Long> {
	Movie getByTitle(String title);
}

File: outgoing-simple-kotlin-demo/src/main/kotlin/de/codecentric/cm4sb/demo/outgoing/repository/MovieRepository.kt
package de.codecentric.cm4sb.demo.outgoing.repository

import de.codecentric.cm4sb.demo.outgoing.domain.Movie
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface MovieRepository : CrudRepository<Movie, Long> {
   fun getByTitle(title: String): Movie
}
